tap_klaviyo/streams.py

```diff
@@ -151,12 +151,12 @@ class ListPersonStream(KlaviyoStream):
     """Define custom stream."""
 
     name = "listperson"
-    path = "/lists/{list_id}/relationships/profiles/"
+    path = "/lists/{list_id}/profiles/"
     primary_keys = ["id"]
     replication_key = None
     parent_stream_type = ListsStream
     schema_filepath = SCHEMAS_DIR / "listperson.json"
-    max_page_size = 1000
+    max_page_size = 100
```

Comment:
The max page size for the profiles endpoint is 10x lower than the relationships one which is making the request be 10x slower.